### PR TITLE
Feat/update to 0.2.0

### PR DESCRIPTION
**Pull request by dmolina79**
Change Log:
* bump dependency to vault 0.2.0
* update Sample Strategy to work with new API updates
* update readme

Screen Shot 2020-11-20 at 08 26 06

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,27 +29,29 @@ TODO
...
+
   - > For those of you using Node 14, you'll need to launch ganache-cli via node 8, 10, or 12 until this is fixed. For example, run nvm use 12 && npm install ganache-cli -g to install for node 12, and then run ganache-cli with nvm use 12 && ganache-cli.
 
 - Install npm dependencies: `npm install`
 
 - Install project dependencies:
+
 ```
-$ brownie pm install iearn-finance/yearn-vaults@0.1.1
+$ brownie pm install iearn-finance/yearn-vaults@0.2.0
```
Comment (fubuloubu) [inline README.md:44]: Come to think of it, I think Brownie handles this internally (on the first install)

File: contracts/Strategy.sol [hunk 2]
```diff
@@ -6,7 +6,10 @@ pragma solidity 0.6.12;
 pragma experimental ABIEncoderV2;
 
 // These are the core Yearn libraries
-import {BaseStrategy, StrategyParams} from "@yearnvaults/contracts/BaseStrategy.sol";
+import {
+    BaseStrategy,
+    StrategyParams
+} from "@yearnvaults/contracts/BaseStrategy.sol";
```
Comment (fubuloubu) [inline contracts/Strategy.sol:12]: Is this a new linter?

**Review by fubuloubu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Macarse) [reply README.md:44]: yep, you always mention pm and I never used it. I just run `brownie compile`

File: README.md [hunk 3]
```diff
@@ -31,27 +29,29 @@ TODO
 - [Install Brownie](https://eth-brownie.readthedocs.io/en/stable/install.html)
 
 - Install Dev dependencies: `pip3 install -r requirements-dev.txt`
-    - Use whatever pip version avaiable
+
+  - Use whatever pip version avaiable
```
Comment (Macarse) [inline README.md:33]: typo `avaiable`

File: contracts/Strategy.sol (hunk 2 above)
Comment (dmolina79) [reply contracts/Strategy.sol:12]: Good question, i thought it was applying it from the project but i guess is using my global config which is similar to the configs from vaults v2 repo, should i undo the style changes or is that good to push?

**Review by Macarse — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dmolina79) [reply README.md:44]: yeah it pulls it for you directly

**Pull request merged**